nightfall-client/client.env

```diff
@@ -0,0 +1,4 @@
+ETH_NETWORK=goerli
+BLOCKCHAIN_URL=your web3 url provider to access the blockchain
```

Comment:
Don't we need a proposer URL?

Reply:
The proposer url now is got in the `nightfall-client` inside `transfer` or `withdraw` when an off-chain transaction. It gets the url from the proposer structure of the contracts.